tags.go

```diff
@@ -213,15 +213,15 @@ func (client *Client) GetTagsForServiceWithAlias(alias string) ([]Tag, error) {
 	if serviceErr != nil {
 		return nil, serviceErr
 	}
-	return client.GetTagsForService(service.Id)
+	return client.GetTagsForService(ID(service.Id))
```

Comment:
why is the extra `ID()` needed here?

Reply:
It shouldn't be needed after I converted service.Id to ID type. Must have forgotten to remove it.